src/Go/effects/netscriptGoImplementation.ts

```diff
@@ -402,7 +401,7 @@ export function resetStats(resetAll = false) {
       };
     }
   } else {
-    Go.stats[GoOpponent.none] = newOpponentStats();
+    Go.prestigeAugmentation();
```

Comment:
This seems like it's going to reset node power and bonuses? For all opponents?

Reply:
good catch. This was from when I was testing a different approach to this that I scrapped, but forgot to roll back in this branch.